language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -44,38 +44,37 @@ const getEnv = (variable: string): string => {
   return result;
 }
 
-const spawnJvmAndWaitOn = async (jar: string, args: string[], resource: string, jvmArgs: string[] = []): Promise<ChildProcess> => {
+const spawnJvm = (jar: string, args: string[], jvmArgs: string[] = []): ChildProcess => {
   const java = getEnv('JAVA');
   const proc = spawn(java, [...jvmArgs, '-jar', jar, ...args], {stdio: "inherit",});
-  await waitOn({resources: [resource]})
```

Comment:
Does moving the `waitOn` to the call site actually change anything?

Reply:
Yes, explained in the PR description.

Reply:
Damn, I always forget to read the PR descriptions. I wish they were at the top of the "Files changed" tab as well just to shove them in my face. Thanks for your explanation and your patience. Also, damn global variables! (I'm painfully aware I'm the one who set this up...)